src/App.vue

```diff
@@ -0,0 +1,234 @@
+<template>
+  <div id="app">
+    <div>
+      <div class="sora_logo">
+        <img src="@/assets/icons/logo.svg" alt="Sora - The Decentralized Autonomous Economy">
+        <span class="sora_logo-text">On this page SoraNet accounts with XOR are listed in descending order.</span>
+      </div>
+      <div class="sora_row">
+        <div class="sora_block sora_total-block">
+          <div class="sora_total-left">
+            <img src="@/assets/icons/xor-icon.svg" class="sora_total-img" alt="XOR Total Supply" />
+          </div>
+          <div class="sora_total-right">
+            <span class="sora_title small">Total Supply</span>
+            <span class="sora_total-amount">{{ Number(this.totalSupply || 0).toLocaleString() }}</span>
+          </div>
+        </div>
+        <div class="sora_block">
+         Last update (every 24hr):<br><b>{{ lastUpdate }}</b>
+        </div>
+      </div>
+      <div class="sora_block sora_table-block">
+        <span class="sora_title">XOR holders in SoraNet</span>
+        <el-table
+          :data="holders"
+        >
+          <el-table-column
+            type="index"
+            width="50"
+            label="#"
+          >
+          </el-table-column>
+          <el-table-column
+            prop="accountId"
+            label="Wallet Account ID"
+          >
+          </el-table-column>
+          <el-table-column
+            prop="amount"
+            label="Amount"
+          >
+          </el-table-column>
+          <el-table-column
+            prop="percentage"
+            label="% of Total Supply">
+          </el-table-column>
+        </el-table>
+      </div>
+    </div>
+    <div class="sora_footer">
+      <span>Sora - The Decentralized Autonomous Economy</span>
+      <a href="https://sora.org/" class="sora_footer-link" alt="Sora.org">Sora.org</a>
+    </div>
+  </div>
+</template>
+
+<script>
+import axios from 'axios'
+import BN from 'bignumber.js'
+import format from 'date-fns/format'
+
+export default {
+  name: 'app',
+  data () {
+    return {
+      URL: process.env.NODE_ENV === 'development'
```

Comment:
Maybe better to store URL as environment variable?

Reply:
Next PR will be about it